forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/DeviceBindAuthenticators.kt

```diff
@@ -96,18 +108,14 @@ interface DeviceAuthenticator {
  * @param keyAlias KeyAlias for
  */
 
-data class KeyPair(
-    val publicKey: RSAPublicKey,
-    val privateKey: PrivateKey,
-    var keyAlias: String
-)
+data class KeyPair(val publicKey: RSAPublicKey, val privateKey: PrivateKey, var keyAlias: String)
 
 /**
  * Settings  for all the biometric authentication is configured
  */
 internal class BiometricOnly(private val biometricInterface: BiometricHandler,
-                    private val keyAware: KeyAware,
-                    private val isApi30OrAbove: Boolean): DeviceAuthenticator {
+                             private val keyAware: KeyAware,
+                             private val isApi30OrAbove: Boolean) : DeviceAuthenticator {
```

Comment:
private val isApi30OrAbove: Boolean) : DeviceAuthenticator - Remove Space
 
like this

private val isApi30OrAbove: Boolean):  DeviceAuthenticator - Remove Space